src/routes/zip.ts

```diff
@@ -12,18 +13,14 @@ export const router = express.Router();
  * Request zipping of files. Require directory:string and files:string[] in the request body
  */
 router.post("/", (req, res) => {
-  console.log("Request body : ", req.body);
   const { hasAccess, statusCode, error, directory, fileNames } = hasFileAccess(
     req,
     req.body.directory,
     req.body.files
   );
-  console.log("Has access  : ", hasAccess);
-  console.log("Status code : ", statusCode);
-  console.log("Error       : ", error);
-  console.log("Directory   : ", directory);
-  console.log("File names  : ", fileNames);
```

Comment:
Maybe we should include the logs like so to prevent spam while keeping access logs?
```
      Logger.log("File accessed", {
        hasAccess: hasAccess,
        statusCode: statusCode,
       error: error,
       directory:directory,
       fileNames:fileNames
      });
```

Reply:
@martin-trajanovski can we log that a request has been submitted to this endpoint?
Also, in the log, could we make sure that we list all the files/datasets requested and the user that requested them?